test/unit/util/mapbox.test.js

```diff
@@ -616,6 +615,248 @@ test("mapbox", (t) => {
 
         t.end();
     });
+    t.test('MapLoadEvent', (t) => {
```

Comment:
I think a test of both events together would also be helpful, since they share state/data.